.github/workflows/continous-integration-os.yml

```diff
@@ -270,4 +270,4 @@ jobs:
         uses: EmbarkStudios/cargo-deny-action@v1
         with:
           command: check ${{ matrix.checks }}
-          args: --workspace with features ${{ matrix.features }}
+          args: --workspace ${{ matrix.features }}
```

Comment:
```suggestion
          arguments: --workspace ${{ matrix.features }}
```

Reply:
Yep, that's the correct syntax:
https://github.com/EmbarkStudios/cargo-deny-action#use-specific-rust-version